eng/packaging.targets

```
@@ -269,4 +269,20 @@
     <Error Condition="'$(ServicingVersion)' == '0'" Text="ServicingVersion is set to 0 and it should be an increment of the patch version from the last released package." />
   </Target>
 
+
+  <!-- For experimental ref assemblies (which typically have the same name as a regular ref
+       assembly), bump their minor file version by 100 to make it distinguishable from the regular
+       ref assembly.
+  -->
+  <Target Name="_ExperimentalRefAssemblyUpdateFileVersion"
```

nit: We don't use lowercase prefixes when a target isn't just an implementation detail:

```suggestion
  <Target Name="UpdateExperimentalRefAssemblyFileVersion"
```